examples/jenkins/master-slave/plugins.txt

```diff
@@ -1,8 +0,0 @@
-git:2.4.2
-git-client:1.19.0
-ssh-credentials:1.11
-mailer:1.16
-matrix-project:1.6
-scm-api:1.0
-script-security:1.13
-junit:1.2
```

Comment:
OK ... merging into the top level README and completely removing master-slave has been pushed